### PR TITLE
Rst export fixes

### PR DESCRIPTION
**Pull request by rduivenvoorde**

### CODE REVIEW

**Comment by jef-n** [comment]
@timlinux how's the analysis going? ;)

**Comment by timlinux** [comment]
Hehe that is some bot doint magick somewhere @jef-n 

@ann26 can you please review, merge, deploy etc?

Thanks!

File: deployment/docker/REQUIREMENTS.txt [hunk 1]
```diff
@@ -34,4 +34,4 @@ reportlab==3.4.0
 xhtml2pdf==0.2b1
 django-modeltranslation
 django-colorfield
-WeasyPrint
+WeasyPrint==0.42.3
```
Comment (ann26) [inline deployment/docker/REQUIREMENTS.txt]: hi @rduivenvoorde the current code in develop is already fixed this. Could you pull the latest code please? Sorry I missed your PR..

**Comment by ann26** [comment]
hi @rduivenvoorde thank you for this PR! Sorry for the long wait. I must have missed this PR. 
@timlinux i will deploy this soon

**Pull request merged**